addon/helpers/consume-context.js

```diff
@@ -2,9 +2,9 @@ import Helper from '@ember/component/helper';
 import { getProviderFor } from '../-private/get-provider-for';
 
 export default class ContextConsumerHelper extends Helper {
-  compute([key]) {
-    const provider = getProviderFor(this, key);
+  compute([key], { fallback }) {
+    const provider = getProviderFor(this, key, fallback);
```

Comment:
Some thoughts here:

I'm wondering if we might want to update the name of `getProviderFor` since now it isn't necessarily always supposed to return a provider anymore if there is a fallback provided?

Also curious if the `assert` should live in this helper now rather than inside `getProviderFor`?

Reply:
> I'm wondering if we might want to update the name of getProviderFor since now it isn't necessarily always supposed to return a provider anymore if there is a fallback provided?

Hmm yeah, that might be a good idea, though I don't mind that it's still called `getProviderFor`. What might you name it instead? 

> Also curious if the assert should live in this helper now rather than inside getProviderFor?

I think it still makes sense that the assert lives in the helper, since the decorator uses this utility as well.

Reply:
Hm yeah the naming thing isn't as big of a deal now that I think about it, I guess it still works the same as it always did. 
 `getProviderFor` is private anyway so it's not like it's a public method that needs to be super specific.